eng/build.yml

```diff
@@ -1,53 +1,54 @@
 parameters:
+  # Agent OS identifier and used as job name
   agentOs: ''
-  queue: {}
+
+  # Agent pool
+  pool: {}
+
+  # Additional variables
   variables: {}
 
-phases:
-- template: /eng/common/templates/phases/base.yml
+  # Build strategy - matrix
+  strategy: ''
+
+  # Job timeout
+  timeoutInMinutes: 120
```

Comment:
It doesn't appear that this property (`timeoutInMinutes`) is being passed through to the `job.yml` template.